composer.json

```diff
@@ -70,6 +73,10 @@
         ],
         "post-create-project-cmd": [
             "@php artisan key:generate --ansi"
+        ],
+        "warmup": [
+            "php artisan config:cache",
+            "php artisan view:cache"
```

Comment:
caching routes led to an application crash. I'll assume that is due to the setup switching logic